binding.c

```diff
@@ -691,6 +691,36 @@ bare_tcp_close (js_env_t *env, js_callback_info_t *info) {
   return NULL;
 }
 
+static js_value_t *
+bare_tcp_keepalive (js_env_t *env, js_callback_info_t *info) {
+  int err;
+
+  size_t argc = 3;
+  js_value_t *argv[3];
+
+  err = js_get_callback_info(env, info, &argc, argv, NULL, NULL);
+  assert(err == 0);
+
+  assert(argc == 3);
+
+  bare_tcp_t *tcp;
+  err = js_get_arraybuffer_info(env, argv[0], (void **) &tcp, NULL);
+  assert(err == 0);
+
+  int32_t keepalive;
+  err = js_get_value_int32(env, argv[1], &keepalive);
```

Comment:
```suggestion
  bool keepalive;
  err = js_get_value_bool(env, argv[1], &keepalive);
```

Reply:
interesting, I was using Integer because the method signature: `int uv_tcp_keepalive(*handle, int enable, unsigned int delay)`, nice to see that the conversion isn't needed

Reply:
`libuv` targets C89 so `bool` isn't available and they use `int` instead.